core/com.b2international.snowowl.core.rest.tests/src/com/b2international/snowowl/core/rest/bundle/BundleRestApiTest.java

```diff
@@ -117,4 +118,100 @@ public void serachBundleByTitle() {
 	}
 	
+	@Test
+	public void expandBundleResources() {
+		final String rootBundleId = "rootBundleId";
+		final String subBundleId = "subBundleId";
+		
+		assertBundleCreated(prepareCreateRequestBody(rootBundleId));
+		assertBundleCreated(prepareCreateRequestBody(subBundleId, rootBundleId));
+		assertCodeSystemCreated(prepareCodeSystemCreateRequestBody("cs1").with("bundleId", rootBundleId));
+		assertCodeSystemCreated(prepareCodeSystemCreateRequestBody("cs2").with("bundleId", rootBundleId));
+		assertCodeSystemCreated(prepareCodeSystemCreateRequestBody("cs3").with("bundleId", subBundleId));
+		
+		assertBundleSearch(Map.of("id", Set.of(rootBundleId), "expand", "resources()")).and()
+			.body("total", equalTo(1))
+			.body("items[0].resources.total", equalTo(3))
+			.body("items[0].resources.items", hasItem(hasEntry("id", "cs1")))
+			.body("items[0].resources.items", hasItem(hasEntry("id", "cs2")))
+			.body("items[0].resources.items", hasItem(hasEntry("id", subBundleId)))
+			.assertThat();
+	}
+	
+	@Test
+	public void updateBundleTitle() {
+		final String id = "b4";
+		assertBundleCreated(prepareCreateRequestBody(id));
+		assertUpdateBundleField(id, "title", "new bundle title");
+	}
+
+	@Test
+	public void updateBundleUrl() {
+		final String id = "b5";
+		assertBundleCreated(prepareCreateRequestBody(id));
+		assertUpdateBundleField(id, "url", "new bundle url");
+	}
+
+	@Test
+	public void updateBundleLanguage() {
+		final String id = "b6";
+		assertBundleCreated(prepareCreateRequestBody(id));
+		assertUpdateBundleField(id, "language", "hu");
+	}
+	
+	@Test
+	public void updateBundleDescription() {
+		final String id = "b7";
+		assertBundleCreated(prepareCreateRequestBody(id));
+		assertUpdateBundleField(id, "description", "Bundle `Hungarian` resources");
+	}
+	
+	@Test
+	public void updateBundleStaus() {
+		final String id = "b8";
+		assertBundleCreated(prepareCreateRequestBody(id));
+		assertUpdateBundleField(id, "status", "draft");
+	}
+	
+	@Test
+	public void updateBundleCopyright() {
+		final String id = "b9";
+		assertBundleCreated(prepareCreateRequestBody(id));
+		assertUpdateBundleField(id, "copyright", "Licensed under the Fictive License 2.0");
+	}
+	
+	@Test
+	public void updateBundleOwner() {
+		final String id = "b10";
+		assertBundleCreated(prepareCreateRequestBody(id));
+		assertUpdateBundleField(id, "owner", "B2i");
+	}
+	
+	@Test
+	public void updateBundleContact() {
+		final String id = "b11";
+		assertBundleCreated(prepareCreateRequestBody(id));
+		assertUpdateBundleField(id, "contact", "info@b2international.com");
+	}
+	
+	@Test
+	public void updateBundleUsage() {
+		final String id = "b12";
+		assertBundleCreated(prepareCreateRequestBody(id));
+		assertUpdateBundleField(id, "usage", "For testing");
+	}
+	
+	@Test
+	public void updateBundlePurpose() {
+		final String id = "b13";
+		assertBundleCreated(prepareCreateRequestBody(id));
+		assertUpdateBundleField(id, "purpose", "Test bundle REST API update endpoint");
```

Comment:
There are no assertions in these tests, as minimum REST API tests should always explicitly check for status codes returned in the response. Could you please add them? Thanks!